<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow_gc</class>
 <widget class="QMainWindow" name="MainWindow_gc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gestion des commandes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tab_mail">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>811</width>
      <height>641</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>ajouter</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>741</width>
        <height>441</height>
       </rect>
      </property>
      <property name="title">
       <string>ajouter commandes</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>70</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>reference:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>110</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>entreprise:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>150</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>objet:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>210</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>type:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>250</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>etat:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>290</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>date:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>330</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>contact:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>370</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>email:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="reference">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>70</y>
         <width>241</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="entreprise">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>110</y>
         <width>241</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="objet">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>150</y>
         <width>241</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="type">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>210</y>
         <width>241</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="etat">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>250</y>
         <width>241</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="contact">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>330</y>
         <width>241</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="Email">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>370</y>
         <width>241</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="modifier">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>130</y>
         <width>171</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ajouter">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>60</y>
         <width>171</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>ajouter</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="date">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>290</y>
         <width>241</width>
         <height>22</height>
        </rect>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QTableView" name="tab_ref">
       <property name="geometry">
        <rect>
         <x>395</x>
         <y>200</y>
         <width>311</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="details">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>130</y>
         <width>161</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>view details</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>190</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>0:vente;1:achat</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>afficher</string>
     </attribute>
     <widget class="QTableView" name="tab_commandes">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>761</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="trie">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>270</y>
        <width>181</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>trie </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rechercher">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="rechercherb">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>360</y>
        <width>151</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>rechercher commande</string>
      </property>
     </widget>
     <widget class="QTableView" name="tab_recherche">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>400</y>
        <width>771</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="PDF">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>270</y>
        <width>181</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>PDF des commandes</string>
      </property>
     </widget>
     <widget class="QPushButton" name="affichercommande">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>221</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>afficher commandes</string>
      </property>
     </widget>
     <widget class="QComboBox" name="trier_gp">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>270</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>date</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>type</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>etat</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>supprimer</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>701</width>
        <height>451</height>
       </rect>
      </property>
      <property name="title">
       <string>supprimer commandes</string>
      </property>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>100</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>reference</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ref">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>91</y>
         <width>261</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="supprimer">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>250</y>
         <width>231</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>historique</string>
     </attribute>
     <widget class="QTableView" name="history">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>711</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="displayhistory">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>430</y>
        <width>181</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>refrech history</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>stat</string>
     </attribute>
     <widget class="QPushButton" name="refrech">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>410</y>
        <width>181</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>refrech</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>581</width>
        <height>331</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="label_10"/>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>mail</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>761</width>
        <height>551</height>
       </rect>
      </property>
      <property name="title">
       <string>envoyer email</string>
      </property>
      <widget class="QLineEdit" name="mentreprise">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>90</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="msujet">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>150</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="sendMail">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>100</y>
         <width>211</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string>envoyer mail</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>90</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>mail entreprise</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>150</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>sujet mail</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>220</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>message</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="mmessage">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>220</y>
         <width>581</width>
         <height>301</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="paswd">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>120</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="uname">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>60</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>username</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>120</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>mot de passe google</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>895</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
